src/modules/storefront/components/Banner.tsx

```diff
@@ -92,12 +92,15 @@ const Slide = ({
       : '';
 
   const overlayProp = `linear-gradient(0deg, rgba(0, 0, 0, 0.5), ${overlayColor})`;
-  const overlayBg = !overlayColor
-    ? bg
-    : mediaType === 'image'
-    ? `${overlayProp}, ${bg}`
-    : overlayProp;
-  const videoClass = mediaType === 'video' && 'pw-absolute';
+
+  const overlayBg =
+    mediaType === 'image'
+      ? `${overlayProp}, ${bg}`
+      : mediaType === 'video'
+      ? overlayProp
+      : bg;
```

Comment:
Esses ternários não ajudam em nada a leitura. É preferível usar um if else no lugar.
```suggestion
 let overlayBg = bg;
if (mediaType === 'image') {
  overlayBg = `${overlayProp}, ${bg}`;
} else if (mediaType === 'video') {
  overlayBg = overlayProp;
}
```